data/json/construction.json

```diff
@@ -889,6 +889,7 @@
         "category" : "CONSTRUCT",
         "required_skills" : [ [ "fabrication", 4 ] ],
         "time" : 180,
+        "tools": [ [ [ "con_mix", 50 ] ] ],
```

Comment:
I think there should be two square brackets, not three, like `[ [ "con_mix", 50 ] ]`.